File: src/Poker/game/PlayerSimple.java
package Poker.game;

public class PlayerSimple
{
   public String name;
   public int chips;
   public boolean inHand;
   public int chipsThisRound;
   public Card[] holeCards;
   
   public PlayerSimple(String name, int chips, boolean inHand, int chipsThisRound)
   {
      this.name = name;
      this.chips = chips;
      this.inHand = inHand;
      this.chipsThisRound = chipsThisRound;
   }
}

File: server/game/playerSimple.js
let PlayerSimple = (name, chips, hasHoleCards, holeCards, ongoingRoundAction) =>
{
	return {
		name: name,
		chips: chips,
		hasHoleCards: hasHoleCards,
		holeCards: holeCards ? holeCards.slice() : null,
		ongoingRoundAction: Object.assign({}, ongoingRoundAction)
	}
}

module.exports = PlayerSimple;
